gazebo_ros/include/gazebo_ros/node.hpp

```diff
@@ -217,5 +224,20 @@ Node::SharedPtr Node::CreateWithArgs(Args && ... args)
 
   return node;
 }
+
+// Class to hold the global set of tracked node names.
+class ExistingNodes
```

Comment:
yeah, I will move the definition to cpp.

Reply:
The definition for this is already in cpp file should I move the class declaration as well to cpp?

Reply:
I think technically, the methods are defined in the cpp file, but the class is still being defined here. I was referring to moving this class definition to the cpp file, though maybe that's not possible since we are defining a member of this type above :thinking:

Reply:
I see. I think write now the class declaration is in header and definition in cpp.